llvm.sh

```diff
@@ -0,0 +1,173 @@
+#!/bin/bash
```

Comment:
Looks like you accidentally included the llvm download script (twice) when you were installing clang-11. Could you please remove them :)

Reply:
yea my bad. I was testing if adding it would have let the jenkins be built but it still isn't working. To remove commits i just do it through the terminal right?

Reply:
You don't need to remove the commit. If you just remove the files (either move them out of the directory, or just delete them), you should be able to just make a new commit with the files gone.